FeatureDetect/src/source/featurelib.h

```diff
@@ -0,0 +1,27 @@
+
+#ifndef FEATUREDETECT_FEATURELIB_H
+#define FEATUREDETECT_FEATURELIB_H
+
+#include <vector>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/objdetect.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/features2d.hpp>
+//#include <opencv2/xfeatures2d.hpp>
+//#include <opencv2/xfeatures2d/nonfree.hpp>
+#include <opencv2/calib3d.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/utility.hpp>
+#include <opencv2/core/ocl.hpp>
```

Comment:
Why so many imports? It seems like they are unused.

Reply:
done